aspnetcore/blazor/fundamentals/signalr.md

```diff
@@ -364,12 +364,14 @@ endpoints.MapBlazorHub(configureOptions: options =>
 });
 ```
 
-Add the following script to the `Shared/_Layout.cshtml` file immediately inside the closing `</body>` tag. WebSockets (`1`) and Long Polling (`4`) are the supported `HTTPTransportTypes`. The following example:
+Add the following script to the `Pages/_Layout.cshtml` file immediately inside the closing `</body>` tag. WebSockets (`1`) and Long Polling (`4`) are the supported `HTTPTransportTypes`. The following example:
 
+* Adds the `autostart="false"` attribute to the Blazor script tag.
 * Specifies support for both WebSockets and Long Polling transports (`1 | 4`).
 * Defaults to the WebSockets transport when a WebSockets connection can be established.
 
 ```html
+<script src="_framework/blazor.server.js" autostart="false"></script>
```

Comment:
Is this doc supposed to be from scratch? I ask because users will likely already have

```html
<script src="_framework/blazor.server.js"></script>
```

and I'm concerned they may add a second 

```html
<script src="_framework/blazor.server.js" autostart="false"></script>
```

which would cause issues.

Reply:
I'll make a small change to the guidance and use the word "existing" above.